database/seed.js

```diff
@@ -14,4 +14,4 @@ for (var i = 0; i < 100; i++) {
 	fakeDataset.push(mockData);
```

Comment:
The spacing looks a bit wide here for the style guidelines. Are you using a linter?